actions/websocket_actions.jsx

```diff
@@ -457,15 +459,15 @@ export function handleEvent(msg) {
         break;
 
     case SocketEvents.PLUGIN_ENABLED:
-        handlePluginEnabled(msg);
+        wrapEventWithJitter(handlePluginEnabled, msg);
```

Comment:
No changes to make here. I just realized that this is a really important one that should've had a jitter here. It's fairly rare that this happens, but I could even see adding more jitter to it in the future since loading plugin JS code is much bigger than regular API responses

Reply:
added override jitter on the function